addon-test-support/@ember/test-helpers/wait-until.ts

```diff
@@ -59,6 +59,7 @@ export default function waitUntil<T>(
           value = callback();
         } catch (error) {
           reject(error);
+          return;
```

Comment:
This was a doosey of a thing to debug 😭. Basically, without this change `waitUntil` would run **forever** if the callback ever threw an error. 😭

This probably needs to be backported to the 1.x branch.